### PR TITLE
v2

### PR DESCRIPTION
**Pull request by josephbuchma**
Upd for https://github.com/db-journey/migrate/pull/3

### CODE REVIEW

File: postgres_test.go [hunk 1]
```diff
@@ -8,15 +8,14 @@ import (
...
 
 // TestMigrate runs some additional tests on Migrate().
 // Basic testing is already done in migrate_test.go
 func TestMigrate(t *testing.T) {
-	host := os.Getenv("POSTGRES_PORT_5432_TCP_ADDR")
-	port := os.Getenv("POSTGRES_PORT_5432_TCP_PORT")
-	driverURL := "postgres://postgres@" + host + ":" + port + "/template1?sslmode=disable"
+	host := getenvDefault("POSTGRES_PORT_5432_TCP_ADDR", "localhost")
+	port := getenvDefault("POSTGRES_PORT_5432_TCP_PORT", "5432")
+	driverURL := "postgres://postgres:p@" + host + ":" + port + "/template1?sslmode=disable"
```
Comment (gravis) [inline postgres_test.go:19]: there's no password when using docker, why this change?

Comment (josephbuchma) [reply postgres_test.go:19]: yeah, that should've been removed.
